Doc/library/string.rst

```diff
@@ -657,9 +657,15 @@ Nesting arguments and more complex examples::
 Template strings
 ----------------
 
-Templates provide simpler string substitutions as described in :pep:`292`.
-Instead of the normal ``%``\ -based substitutions, Templates support ``$``\
--based substitutions, using the following rules:
+Template strings provide simpler string substitutions as described in
+:pep:`292`.  The primary use case for template strings is for
+internationalization (i18n) since in those contexts, the more featured yet
+complex formatting languages for f-strings and ``str.format()`` can make
+translations more difficult to implement.  As an example of a library for i18n
```

Comment:
And f-strings can't be used for internationalization at all. This is a syntax, not format string. printf-like formatting can be mentioned. It is used for internationalization and is more powerful and more complex than Template.